backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/BitcodeCompiler.kt

```diff
@@ -103,9 +103,17 @@ internal class BitcodeCompiler(val context: Context) {
 
         val profilingFlags = llvmProfilingFlags().map { listOf("-mllvm", it) }.flatten()
 
+        // LLVM Pipeline that we use know nothing about arm64_32 so we override it here.
```

Comment:
LLVM Pipeline that we use know nothing about - LLVM we use does not have support for arm64_32. TODO: fix with LLVM update.